search/includes/classes/class-search.php

```diff
@@ -63,6 +63,74 @@ class Search {
 		'advanced_seo_description',
 	);
 
+	public const POST_PROTECTED_TAXANOMIES = array(
```

Comment:
Probably a spelling error? TaxOnomies

Reply:
Question: are these really taxonomies? They look like query parameters to me.

Reply:
Yeah probably not the best name. 

The issue is that those are kinda reserved keywords in a request towards elasticsearch. If you would create a custom taxanomy with the same search name and query for it, it would overwrite these keywords like `fields`.

Any suggestions on how to name it?

Reply:
Maybe a variation of RESERVED_NAMES? Summoning @rinatkhaziev, as his English-fu is much better than mine

Reply:
How does `ES_QUERY_RESERVED_NAMES` sound?